3.7/glacier2/Dockerfile

```diff
@@ -1,18 +1,35 @@
-FROM ubuntu:16.04
+FROM ubuntu:22.04
 
 LABEL maintainer="docker-maintainers@zeroc.com"
 
-ARG GLACIER2_VERSION=3.7.8
-
+ARG GLACIER2_VERSION=3.7.9
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv B6391CB2CFBA643D \
-    && echo "deb http://download.zeroc.com/Ice/3.7/ubuntu16.04 stable main" >> /etc/apt/sources.list.d/ice.list \
-    && apt-get update \
+RUN groupadd -r ice && useradd -r -g ice ice
```

Comment:
Is this required, I thought the packages already do that https://github.com/zeroc-ice/ice-debian-packaging/blob/a9e4bde1930bb659f9c50e6372092917428c722e/debian/zeroc-glacier2.glacier2router.service#L17

Reply:
yes, but this is a Docker best practice to ensure a constant id is created across rebuilds.